Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -86,13 +88,15 @@ class TopSitesPanel: UIViewController {
 
     var profile: Profile! {
         didSet {
-            self.refreshHistory(self.layout.thumbnailCount)
```

Comment:
If I'm reading the code right, the profile is provided exactly once, immediately after construction (`HomePanels.swift`).
And once set it doesn't change.

So can we achieve the same benefit by having `let profile` and setting it in the initializer, and eliminate this `didSet` altogether?